package/samplers/nsgaii_with_initial_trials/README.md

```diff
@@ -36,12 +52,14 @@ def objective(trial: optuna.Trial) -> tuple[float, float]:
 
 
 storage = optuna.storages.InMemoryStorage()
+study_name = "test"
+directions=["minimize", "minimize"],
```

Comment:
`["minimize", "minimize"],` results in the tuple `(["minimize", "minimize"],)` and I think it is not intended.

```suggestion
directions=["minimize", "minimize"]
```

Reply:
Thanks for the comment.
It was a typo and has been corrected.